src/layouts/Folders.jsx

```diff
@@ -72,63 +74,138 @@ const Folders = ({ match, location }) => {
           }
         }
       },
-    );
+    )
 
-    const { mutate: rearrangeFolder } = useMutation(
-      payload => setDirectoryFile(siteName, folderName, payload),
+    // parse contents of current folder directory
+    useEffect(() => {
+      if (folderContents && folderContents.data) {
+        const parsedFolderContents = parseDirectoryFile(folderContents.data.content)
+        setDirectoryFileSha(folderContents.data.sha)
+        setParsedFolderContents(parsedFolderContents)
+
+        if (subfolderName) {
+          const subfolderFiles = retrieveSubfolderContents(parsedFolderContents, subfolderName)
+          if (subfolderFiles.length > 0) {
+            setFolderOrderArray(subfolderFiles)
```

Comment:
Rebase issue, should be 
`setFolderOrderArray(subfolderFiles.filter(item => item.name !== '.keep'))`

Reply:
5213f6a